modules/RTC/TraceablePeerConnection.js

```diff
@@ -259,7 +492,8 @@ var normalizePlanB = function(desc) {
                 });
             }
 
-            if (typeof mLine.ssrcs !== 'undefined' && Array.isArray(mLine.ssrcs)) {
+            if (typeof mLine.ssrcs !== 'undefined'
```

Comment:
Can't the first part of the test be deleted? `Array.isArray(undefined)` will be false anyway.